presentation/README.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: pattern
+title: Presentation
+folder: presentation
+permalink: /patterns/presentation/
+categories: Behavioral
+tags:
+ - Decoupling
+---
+## Also known as
+Application Model
+
+## Intent
+Presentation Model pulls the state and behavior of the view out into a model class that is part of the presentation. The Presentation Model coordinates with the domain layer and provides an interface to the view that minimizes decision making in the view. The view either stores all its state in the Presentation Model or synchronizes its state with Presentation Model frequently
+
+## Explanation
```

Comment:
Please provide a proper explanation with the code example that you have already implemented in the README as well.

Reply:
Ok, I will finish it soon.

Reply:
Please use the same format in the explanation as the other patterns do. See for example https://github.com/iluwatar/java-design-patterns/tree/master/abstract-factory

Reply:
The code example is a bit too long currently. Maybe we can show only the relevant parts? In the README.md it's possible to leave out the methods and comments that don't add to the explanation.

Reply:
Think about it as a short blog that explains how to implement the pattern